### DISCUSSION TITLE
Video resolution n M1

### DISCUSSION THREAD

**Discussion by waldorf-statler** [GerryDazoo/Slinger#283] [General]
I realize there have been some discussions around this, but even with search I am finding it difficult to go back to these. Basically I am finding that the video is producing 1280x540 as reported by VLC using the config.ini default settings (resoluion 12). I have tried a few other values but all I can get is lower resolution. The input is component, supposedly HD. In the past I could definitely get higher quality video via Sling software, although to be fair I dont know for sure the detail (all I know is that visually it was clearly higher quality).

Any advice or guidance? Happy to do some reading if pointed in the right direction too. Thanks!

**Comment by GerryDazoo** [comment]
What kind of box is it? Try giving it a 1280x720 input. I've noticed on some boxes, if they're getting a 1920x1080 input it will just throw away half the scan lines to get a lower resolution. Or change the resolution in the config.ini file to 16. Not sure what you mean by "visually higher quality". My code just forwards the same video stream  from the slingbox that slingplayer would get. Here's the stream info from my M1

Input #0, asf, from 'http:/127.0.0.1:8084/slingbox':
  Metadata:
    title           : Slingbox
    artist          : Sling Media
    comment         : test
  Duration: N/A, start: 0.000000, bitrate: 64 kb/s
  Stream #0:0: Video: h264 (h264 / 0x34363268), none, **1280x720**    nan    :  0.000 fd=   0 aq=    0KB vq=    0KB sq=    0, 1k tbr, 1k tbn
  Stream #0:1: Audio: aac (LC) ([255][0][0][0] / 0x00FF), 32000 Hz, stereo, fltp, 64 kb/s

My Config.ini

[SLINGBOX]
sbtype="350/500/M1"
password=blahblahblah 
ipaddress=192.168.117.196
port=5301
Resolution=12
FrameRate=60
VideoBandwidth=2000
VideoSource = 1

**Reply by waldorf-statler** [reply]
Thanks @GerryDazoo that's very helpful. Since you ask, the input is from a sat box that can generate 1080i or 720i, and then goes through one of those splitters with HDMI and component outputs that seemed to be Slingbox specific (I am in the UK, but the splitter was bought in the US). 

Anyway I had the frame rate set to 30, but once this was changed to 60, I could get 1920x1080 in some configs. (e.g. 1920x1080 @ 16, 1280x540 @ 8,12, 15 etc, 640x480 @ 17, 18, 19, 20 etc, even 320x240 @ 13, 14). But a reasonable compromise seems to be to set the box output at 720i - just as you advised - when I am getting 1280x720 @ 12 in VLC. Curious how the resolution index impacts the video - there does not seem to be a logic that I can detect.

**Comment by mets18** [comment]
I have noticed a slight error in the comments in the config.ini file for video source for the M1.  0 is composite and 1 is component.

**Reply by GerryDazoo** [reply]
I just fixed that minutes ago.

**Reply by mets18** [reply]
I still see the M1 grouped with the solo instead of the M2

**Reply by RandomMan719** [reply]
I also don't see video source setup for the Pro, I don't know what it should be, though.  I will have to pull mine out and set it up, sometime and see the sources it takes, and the order of them.

**Reply by sp222222** [reply]
zero is tuner  and 3 is component on PRO-HD
